include/bl_storage.h

```diff
@@ -46,17 +61,54 @@ extern "C" {
  *  This works as ASSEMBLY implies the OTP to be erased.
  */
 struct life_cycle_state_data {
-	uint16_t provisioning;
-	uint16_t secure;
+	lcs_data_t provisioning;
+	lcs_data_t secure;
+#ifdef CONFIG_NRFX_NVMC
```

Comment:
Not sure if it's a good idea to remove the padding. It's not needed for padding purposes, but will be needed if another state is added. Maybe @Vge0rge can chime in.

Reply:
Maybe worth to add reserved for future usage field for RRAM based device as well then.
Just in case.

Reply:
I agree that it's better to have a reserve word there in case we want to support another state later.